test/TestProjects/MgmtMockAndSample/src/Customize/ApplicationRule.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+using MgmtMockAndSample;
+
+namespace MgmtMockAndSample.Models
+{
+    [CodeGenSerialization(nameof(NewIntSerializeProperty), "newIntSerializeProperty")]
+    [CodeGenSerialization(nameof(NewGeneratedTypeSerializeProperty), "newGeneratedTypeSerializeProperty")]
+    public partial class ApplicationRule : FirewallPolicyRule
```

Comment:
we also need to test this functionality in a project that enables model-reader-writer (short forMRW)
or
since in the near future, model-reader-writer will be enabled by default and disabling it will not be an option, I suggest we enable model-reader-writer for this project.

MRW can be enabled by adding a configuration
```
use-model-reader-writer: true
```
in the according `autorest.md`

Reply:
verified the feature with use-model-reader-writer: true and the result is positive. But I would prefer not to include the change of the config in this PR which will introduce change in hundreds of test files which is not related to this feature. thanks.

Reply:
If so, please add the test to another project that enables MRW

Reply:
test added in MgmtCustomization